.gitignore

```diff
@@ -8,3 +8,4 @@ npm-debug.log
 .jshintrc
 .vs/
 test/unit/three.*.unit.js
+.package-lock.json
```

Comment:
Note it's only moved in this PR. Maybe worth filing a separate issue? I don't have a strong preference here but threejs has only devDependencies; there are no prod dependencies to resolve.

Reply:
I ignore this file because in many times, developers might use npm mirror. In my company, we have a mirror so we install form mirror instead of npm website because it will be faster. And I know many companies have their own mirror like Alibaba and Baidu. If we use mirror, the package-lock file is total different. And we know that three.js doen't depend on any third party library and we use npm just install develop tools like babel, eslint, rollup etc. In that way, I think ignore package-lock file is a better choice. If we commit that file, there might be a lot of conflicts in package-lock